cmd/kubeadm/app/apis/kubeadm/v1alpha3/defaults_windows.go

```diff
@@ -20,3 +20,6 @@ package v1alpha3
 
 // DefaultCACertPath defines default location of CA certificate on Windows
 const DefaultCACertPath = "C:/etc/kubernetes/pki/ca.crt"
```

Comment:
let's change this to:
`C:\etc\kubernetes\pki\ca.crt` inside " ` " quotes

or to:
`"C:\\etc\\kubernetes\\pki\\ca.crt"`

i think i found a potential issue with Go's filepath on Windows...grumble.

Reply:
This is not part of this PR I believe.

Reply:
This doesn't logically belong to this PR. I can change it in another PR if needed.